db/migrate/20170619161514_add_options_to_ext_management_system.rb

```diff
@@ -0,0 +1,37 @@
+class AddOptionsToExtManagementSystem < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    has_many :custom_attributes,
+             :as         => :resource,
+             :dependent  => :destroy,
+             :class_name => 'AddOptionsToExtManagementSystem::CustomAttribute'
+    serialize :options
+  end
+
+  class CustomAttribute < ActiveRecord::Base
+    belongs_to :resource, :polymorphic => true
+    serialize :serialized_value
+  end
+
+  def up
+    add_column :ext_management_systems, :options, :text
+    say_with_time("Migrating Kubernetes provider options") do
+      ExtManagementSystem
+        .where(:type => ["ManageIQ::Providers::Kubernetes::ContainerManager",
+                         "ManageIQ::Providers::Openshift::ContainerManager"])
+        .each do |cp|
+        migrated_options = Hash[
+          cp.custom_attributes.where(:section => 'cluster_settings',
+                                     :name    => %w(no_proxy http_proxy https_proxy))
+            .map { |ca| ["image_inspector_#{ca.name}".to_sym, ca.value] }]
+        cp.update(:options => migrated_options)
```

Comment:
the custom attribute names are cluster_settings/no_proxy  cluster_settings/http_proxy etc and the new one is image_inspector_no_proxy ?
I think that would be better expressed with nesting.